tests/ops/test_qubit_ops.py

```diff
@@ -69,6 +70,46 @@
     ),
 ]
 
+EIGVALS_TEST_DATA_MULTI_WIRES = [
+    (
+        functools.reduce(np.kron, [Y, I, Z]),
+        np.array(
+                [-1., -1., -1., -1.,  1.,  1.,  1.,  1.]
+        ),
+        (     [[-0.70710678+0.j, -0.+0.j,
+              0. +0.j ,  0. +0.j ,
+              0. +0.j ,  0. +0.j ,
+              0. +0.j , -0.70710678+0.j ],
+                   [ 0. +0.j , -0. +0.j ,
+              0. -0.70710678j,  0. +0.j ,
+              0. +0.70710678j,  0. +0.j ,
+              0. +0.j ,  0. +0.j ],
+                   [ 0. +0.j , -0.70710678+0.j ,
+              0. +0.j ,  0. +0.j ,
+              0. +0.j , -0.70710678+0.j ,
+              0. +0.j ,  0. +0.j ],
+                   [ 0. +0.j ,  0. +0.j ,
+              0. +0.j ,  0. -0.70710678j,
+              0. +0.j ,  0. +0.j ,
+              0. +0.70710678j,  0. +0.j ],
+                   [ 0. +0.70710678j,  0. +0.j ,
+              0. +0.j ,  0. +0.j ,
+              0. +0.j ,  0. +0.j ,
+              0. +0.j ,  0. -0.70710678j],
+                   [-0. +0.j ,  0. +0.j ,
+              0.70710678+0.j ,  0. +0.j ,
+              0.70710678+0.j ,  0. +0.j ,
+              0. +0.j ,  0. +0.j ],
+                   [-0. +0.j ,  0. +0.70710678j,
+              0. +0.j ,  0. +0.j ,
+              0. +0.j ,  0. -0.70710678j,
+              0. +0.j ,  0. +0.j ],
+                   [-0. +0.j , -0. +0.j ,
+              0. +0.j ,  0.70710678+0.j ,
+              0. +0.j ,  0. +0.j ,
+              0.70710678+0.j ,  0. +0.j ]])
+    ),
+]
```

Comment:
How was the above calculated?

Reply:
It was the output of ``np.linalg.eigh``

Reply:
Ah, I'm curious if it should be hardcoded into the test, since it seems a bit redundant to parametrize with only a single data point. E.g., have
```python
observable = functools.reduce(np.kron, [Y, I, Z])
eigvals, eigvecs = np.linalg.eigh(observable)
```
directly in the test. It would also help make the test a lot more readable